### PR TITLE
feat(FOROME-588): changed format of the Anfisa version

### PR DESCRIPTION
**Pull request by AlMaQntr**

### CODE REVIEW

**Review by AndreyUstyumenko — APPROVED** [review]

File: src/components/header.tsx [hunk 1]
```diff
@@ -105,8 +105,10 @@ export const Header = observer(
               <Logo mode="white" className="mr-4" />
 
               <span className="text-grey-blue">
-                Anfisa front: {process.env.REACT_APP_VERSION} back:{' '}
-                {toJS(dirinfoStore.dirinfo).version as string}
+                {t('header.version', {
+                  frontend: process.env.REACT_APP_VERSION,
+                  backend: toJS(dirinfoStore.dirinfo).version,
+                })}
```
Comment (AndreyUstyumenko) [inline src/components/header.tsx:111]: According to the [task](https://quantori.atlassian.net/browse/FOROME-588), they want the versions to be displayed in a column. In your version they are displayed in a row.

Comment (AlMaQntr) [reply src/components/header.tsx:111]: Changed the layout to columns

Comment (AndreyUstyumenko) [reply src/components/header.tsx:111]: They still displayed in a row.
Can you display them like this? And give them font-size 13px, in order not to bring too much attention to them
Screenshot 2022-01-31 at 23 21 09

Comment (AlMaQntr) [reply src/components/header.tsx:111]: image

Right now, it displayed in a column

**Review by Lapk1n — APPROVED** [review]

**Review by AndreyUstyumenko — APPROVED** [review]

**Pull request merged**